core/runtime/src/main/resources/META-INF/quarkus-extension.yaml

```diff
@@ -12,4 +12,9 @@ metadata:
   categories:
     - "integration"
     - "messaging"
-  status: "preview"
+  codestart:
+    name: flow
```

Comment:
```suggestion
    name: Quarkus Flow
```

Reply:
@ricardozanini it is the name of the codestart, the codestart docs recommends (not recommend, but shows) the name of extension in lowercase (in our case flow).

Reply:
the name of the extension is  `quarkus flow`, not `flow`.